### PR TITLE
Starting Jetpack 12.0-a.8

### PR DESCRIPTION
**Pull request by zinigor**
Fixes #

## Proposed changes:

*

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to '..'
*

### CODE REVIEW

File: projects/js-packages/api/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ### This is a list detailing changes for the Jetpack RNA Components package releases.
 
+## 0.15.3 - 2023-03-20
+
```
Comment (sdixon194) [inline projects/js-packages/api/CHANGELOG.md]:
```suggestion
- Minor internal updates.
```

File: projects/js-packages/connection/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ### This is a list detailing changes for the Jetpack RNA Connection Component releases.
 
+## 0.25.4 - 2023-03-20
```
Comment (sdixon194) [inline projects/js-packages/connection/CHANGELOG.md:5]:
```suggestion
## 0.25.4 - 2023-03-20
- Minor internal updates.
```

File: projects/packages/blaze/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.6] - 2023-03-20
+### Changed
+- Updated package dependencies. [#29471]
+- Updated package dependencies. [#29480]
```
Comment (sdixon194) [inline projects/packages/blaze/CHANGELOG.md]:
```suggestion
```

File: projects/packages/connection/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.51.2] - 2023-03-20
+### Changed
+- Updated package dependencies. [#29471]
+- Updated package dependencies. [#29480]
```
Comment (sdixon194) [inline projects/packages/connection/CHANGELOG.md]:
```suggestion
```

File: projects/packages/forms/CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.0] - 2023-03-20
+### Added
+- Jetpack Forms: Add tracking of Google Sheets exports [#29225]
+
+### Changed
+- Rollback rename of columns/fields on export [#29448]
+- Updated package dependencies. [#29471]
+- Updated package dependencies. [#29480]
```
Comment (sdixon194) [inline projects/packages/forms/CHANGELOG.md]:
```suggestion
```

File: projects/packages/identity-crisis/CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.40] - 2023-03-20
+### Changed
+- Updated package dependencies. [#29471]
+- Updated package dependencies. [#29480]
```
Comment (sdixon194) [inline projects/packages/identity-crisis/CHANGELOG.md]:
```suggestion
```

File: projects/packages/lazy-images/CHANGELOG.md [hunk 7]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.1.34] - 2023-03-20
+### Changed
+- Updated package dependencies. [#29471]
+- Updated package dependencies. [#29480]
```
Comment (sdixon194) [inline projects/packages/lazy-images/CHANGELOG.md]:
```suggestion
```

File: projects/packages/publicize/CHANGELOG.md [hunk 8]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.22.0] - 2023-03-20
+### Added
+- Added SIG image to the post media array if it's enabled [#29093]
+
+### Changed
+- Updated package dependencies. [#29471]
+- Updated package dependencies. [#29480]
```
Comment (sdixon194) [inline projects/packages/publicize/CHANGELOG.md]:
```suggestion
```

File: projects/packages/search/CHANGELOG.md [hunk 9]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.34.0] - 2023-03-20
+### Changed
+- Updated package dependencies. [#29471]
+- Updated package dependencies. [#29480]
```
Comment (sdixon194) [inline projects/packages/search/CHANGELOG.md]:
```suggestion
```

File: projects/plugins/mu-wpcom-plugin/CHANGELOG.md [hunk 10]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.0.11-beta - 2023-03-20
```
Comment (sdixon194) [inline projects/plugins/mu-wpcom-plugin/CHANGELOG.md]: Looks like we are releasing a beta of the mu-wpcom-plugin here. We can just release the next stable version, 1.0.11.

File: projects/plugins/mu-wpcom-plugin/package.json [hunk 11]
```diff
@@ -1,7 +1,7 @@
 {
 	"private": true,
 	"name": "@automattic/jetpack-mu-wpcom-plugin",
-	"version": "1.0.11-alpha",
+	"version": "1.0.11-beta",
```
Comment (samiff) [inline projects/plugins/mu-wpcom-plugin/package.json]: We've been doing stable releases for this plugin, not any beta cycles. Did you select `stable` when running the changelogger for this one?

Comment (zinigor) [reply projects/plugins/mu-wpcom-plugin/package.json]: Whoops, I used a `-b` switch here.

File: projects/plugins/mu-wpcom-plugin/CHANGELOG.md [hunk 12]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.0.11 - 2023-03-20
+### Changed
+- Internal updates.
+
+## 1.0.11-beta - 2023-03-20
```
Comment (sdixon194) [inline projects/plugins/mu-wpcom-plugin/CHANGELOG.md]:
```suggestion
```

**Review by sdixon194 — APPROVED** [review]

**Pull request merged**